google-cloud-storage/src/test/java/org/jclouds/googlecloudstorage/features/ObjectApiLiveTest.java

```diff
@@ -422,8 +425,9 @@ public void testDeleteObjectWithOptions() {
 
    @AfterClass
    private void deleteBucket() {
-      api.getBucketApi().deleteBucket(BUCKET_NAME);
-      api.getBucketApi().deleteBucket(BUCKET_NAME2);
+      assertTrue(api.getBucketApi().deleteBucket(BUCKET_NAME));
+      assertTrue(api.getBucketApi().deleteBucket(BUCKET_NAME2));
+      assertFalse(api.getBucketApi().deleteBucket(NONEXISTENT_BUCKET_NAME));
```

Comment:
I think just removing asserts would be enough. Delete bucket tests are available in BucketApiLiveTests